base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp

```diff
@@ -643,6 +644,7 @@ Result<cvd::Response> CvdStartCommandHandler::Handle(
 
   CF_EXPECT(UpdateArgs(subcmd_args, group));
   CF_EXPECT(UpdateEnvs(envs, group));
+  const auto bin = CF_EXPECT(FindStartBin(group.HostArtifactsPath()));
```

Comment:
[`FindStartBin`](https://github.com/google/android-cuttlefish/blob/19709a6a16df349034cc4426f2e8081a2386b66c/base/cvd/cuttlefish/host/commands/cvd/server_command/start.cpp#L483) suggests running `lunch` if it can't find the executable, does that still help?

Reply:
Done